keps/sig-node/2727-grpc-probe/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Add gRPC probe to Pod.Spec.Container.{Liveness,Readiness,Startup}Probe
+kep-number: 2727
+authors:
+  - "@bowei"
+  - "@PxyUp"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-network
+status: implementable
+creation-date: 2020-04-04
+modified-date: 2021-05-12
+reviewers:
+  - "@thockin"
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@thockin"
+see-also:
+replaces:
+prr-approvers:
```

Comment:
Don't actually need this here anymore, the one in the other file replaces it. But you do need: `stage` and `latest-milestone`. Not sure if you need the overall `milestone` block too.

Reply:
fixed.